TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/RenderViewHelper.php

```diff
@@ -101,9 +137,34 @@ public function render($section = null, $partial = null, $arguments = array(), $
     }
 
     /**
+     * Loads the "argument" child nodes.
+     *
+     * @param $arguments
```

Comment:
Yes, that's even better but the type hint is missing in `loadSettingsIntoArguments()` too, so I didn't complain ;)